scripts/firedrake-install

```diff
@@ -407,30 +422,33 @@ If you really want to use your own Python packages, please run again with the
 
 petsc_opts = "--download-eigen=%s/src/eigen-3.3.3.tgz " % firedrake_env
```

Comment:
This gets ignored completely. Not that it makes a huge difference right now since there are still issues with using Eigen and complex-mode Firedrake together. Eigen not installing when running complex-mode is one of them.